contrib/mark3labs/mcp-go/README.md

```diff
@@ -0,0 +1,36 @@
+# MCP-Go Integration
+
+This integration provides Datadog tracing for the [mark3labs/mcp-go](https://github.com/mark3labs/mcp-go) library.
+
+## Usage
+
+```go
+import (
+    mcpgo "github.com/DataDog/dd-trace-go/contrib/mark3labs/mcp-go/v2"
+    "github.com/DataDog/dd-trace-go/v2/ddtrace/tracer"
+    "github.com/mark3labs/mcp-go/server"
+)
+
+func main() {
+    tracer.Start()
+    // Cleanup internal caches
+    defer tracer.Stop()
+
+    // Create Datadog hooks
+    ddHooks := mcpgo.NewHooks()
+    defer ddHooks.Stop()
+
+    // Add tracing to your existing hooks object
+    hooks := &server.Hooks{}
+    ddHooks.AddHooks(hooks)
```

Comment:
I feel from this line the api usage might look a little bit confusing (we call `ddHooks.AddHooks(something)` which I would understand as "add `something` into `ddHooks`, but its actually the opposite).

I was thinking maybe having a single `mcpgotrace.AddServerHooks(hooks)` function that returns a `cleanup` function would be a little bit more clear, WDYT?